src/main/java/org/motechproject/newebodac/web/EnrollmentController.java

```diff
@@ -0,0 +1,42 @@
+package org.motechproject.newebodac.web;
+
+import java.util.UUID;
+import org.motechproject.newebodac.service.EnrollmentService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@Controller
+public class EnrollmentController extends BaseController {
+
+  @Autowired
+  private EnrollmentService enrollmentService;
+
+  @RequestMapping(value = "/enrollVisit/{id}", method = RequestMethod.PUT)
```

Comment:
can you change it /visit/enroll/{id}? and similarly for the rest of the endpoints?